android_16/resources/products/products_api.py

```diff
@@ -0,0 +1,14 @@
+from flask.views import MethodView
+
+from android_16.resources.products.product import Product
+from android_16.resources.products.product_reader import ProductReader
+
+
+class ProductsApi(MethodView):
+    @staticmethod
+    def get():
+        resp = [Product.serialize(item) for item in ProductReader.get_products()]
```

Comment:
This logic should be handled by something other than the API. Create Api Actor class

Reply:
Moved logic